vllm/config.py

```diff
@@ -459,11 +448,6 @@ def _verify_quantization(self) -> None:
                     "Using AWQ quantization with ROCm, but VLLM_USE_TRITON_AWQ"
                     " is not set, enabling VLLM_USE_TRITON_AWQ.")
                 envs.VLLM_USE_TRITON_AWQ = True
```

Comment:
I think it may be better for each platform to have its own `verify_quantization` method so we can also move this over to rocm platform.

Reply:
It's more clear, thanks.